specs/global_navigation.adoc

```diff
@@ -0,0 +1,82 @@
+# Initial UI refresh for the grafana integration
+
+Since we are having Grafana with Tendrl, the global menu has to be refined. This spec talks about the modules which we can keep from our previous build (it is available https://github.com/Tendrl/ui/tree/ui-legacy-code[ui-legacy-code] ).
+
+## Problem Description
+
+It descibes the items to be retained/added in new global menu
+  
+## Use Cases
+
+Every tendrl user will be able to see this navigation
+
+## Proposed change
+
+Currently the navigation has following items:-
+
+* Dashboard
+* Clusters
+* Hosts
+* Volumes
+* Pools
+* RBDs
+* Alerts
+* Admin
+
+
+The following changes has been done in the navigation
+
+* Removing *Cluster specific objects* items such as *Volumes*, *Pools* and *Rbds*.
+* Retaining items like *Clusters*, *Hosts*, *Admin(tasks)*, *Alerts*
+
+** Cluster List ```module: cluster```
+** Hosts list ```module: host```
+** Tasks list ```module: tasks```
+** Task details page ```module: tasks/task-detail```
+** Global alerts list ```module: alerts```
```

Comment:
Not sure if this is specifically the order in which these items would be shows. Assuming that this list is ordered, I'd figure the following to be more appropriate:

* Clusters
* Hosts
* Alerts
* Admin
  * Tasks
  * User management
  * Notifications configuration etc.